salt/modules/boto_sns.py

```diff
@@ -178,6 +178,30 @@ def subscribe(topic, protocol, endpoint, region=None, key=None, keyid=None, prof
     return True
 
 
+def unsubscribe(topic, subscription_arn, region=None, key=None, keyid=None, profile=None):
+    '''
+    Unsubscribe a specific SubscriptionArn of a topic.
+
+    CLI example to unsubscribe a subscription to a topic::
```

Comment:
Instead of using the `::` syntax here, can you change this to:
```
CLI Example:

.. code-block:: bash

    salt myminion ....
```
This makes our "bash example" code formatting work in our docs.